packages/core/src/table/table.scss

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import './../styles/tokens/generated/index';
+
+:root {
+  --cds-alias-object-color-flat: #{$cds-global-color-gray-0};
+  --cds-alias-object-border-radius-50: 0.1rem; // 2px, existing radius on table
+}
+
+[cds-theme~='dark'] {
+  --cds-alias-object-color-flat: #{$cds-global-color-construction-1000};
+}
+
+$cds-alias-table-color-background: var(--cds-alias-object-color-flat);
+$cds-alias-object-border-radius-50: var(--cds-alias-object-border-radius-50);
+
+// Need to get typography colors correct
+
+// Styles and default bordering
+[cds-table] {
+  background-color: $cds-alias-table-color-background;
+  border-collapse: separate;
+  border-spacing: 0;
+  color: #{$cds-global-color-gray-700};
```

Comment:
Using `$cds-global-typography-color-400` or 300 will get the color contrast and dark theme contrast automatically